google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1904,6 +1927,46 @@ def iter(self):
                 yield FACTORIES['gsuite_group_member'].create_new(data)
 
 
+class ProjectSinkIterator(ResourceIterator):
+    """The Resource iterator implementation for Project Sink"""
+
+    def iter(self):
+        """Yields:
+            Resource: Sink created
+        """
+        gcp = self.client
+        if self.resource.enumerable():
+            for data in gcp.iter_project_sinks(
+                    projectid=self.resource['projectId']):
+                yield FACTORIES['sink'].create_new(data)
+
+
+class FolderSinkIterator(ResourceIterator):
+    """The Resource iterator implementation for Folder Sink"""
+
+    def iter(self):
+        """Yields:
+            Resource: Sink created
+        """
+        gcp = self.client
+        folderid = self.resource['name'].split('/', 1)[-1]
```

Comment:
Should we put this logic into the get_name method in the foldersinkrepos instead?

Reply:
Looks like we don't even need this logic since LoggingFoldersSinksRepository.get_name just adds the prefix back anyway